### PR TITLE
new core web vitals guide

### PR DESCRIPTION
**Pull request by ghost**
Added a new .mdx file and a new .yml nav path to the guide.

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Guide to core web vitals
+tags:
+  - Browser
+  - Browser monitoring
+  - Guides
+metaDescription: "A guide to monitor Google's core web vitals."
+
```
Comment (roadlittledawn) [inline src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx:8]: i _think_ you may need to remove this empty line to fix the error in the build

File: src/content/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring.mdx [hunk 2]
```diff
@@ -38,17 +38,11 @@ Real user monitoring (RUM) gives insights into actual page performance, but you
...
-The [browser monitoring **Summary** page](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) shows important core web vital scores for:
-
-* Largest contentful paint (LCP): how quickly your page content loads
-* First input delay (FID): how quickly the browser can respond to user input
-* Cumulative layout shift (CLS): how unstable is the content being loaded
-
-You can also use the [`PageViewTiming` event](/attribute-dictionary/?event=PageViewTiming) for a more real-time delivery mechanism that does not have dependencies on any other event.
+Google's core web vitals are three metrics to gauge how users experience your browser app - and New Relic can help you track them! There are two places to see core web vitals in New Relic:
+* The [**Summary** page](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) shows core web vitals for your entire site.
+* The [**Web vitals** page](/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals#monitor) shows a breakdown of core web vitals by page URL, device type, location, and more.
```
Comment (nbaenam) [inline src/content/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring.mdx:45]: I think that 'page' should go out of the bolded in both cases or maybe remove the word page. What about one of these 2 options?:

- The [**Summary**](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) page shows....
- [**Summary**](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) shows....

File: src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx [hunk 3]
```diff
@@ -0,0 +1,185 @@
+---
+title: Monitor core web vitals
+tags:
+  - Browser
+  - Browser monitoring
+  - Guides
+metaDescription: "A guide to monitor Google's core web vitals."
+---
+
+import browserWebVitalsGif from 'images/browser_screenshot-full_web-vitals-gif.gif'
```
Comment (nbaenam) [inline src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx:10]: I think the '-gif.' mustn't be here. Maybe the macro didn't go well: browser_screenshot-full_web-vitals-gif.gif

File: src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+import browserFilteredWebVitals from 'images/browser_screenshot-full_filtered-web-vitals.png'
+
+Core web vitals are Google’s metrics to gauge overall user experience of your site, which can influence your site’s SEO rankings and give you valuable insight into how users perceive your business.  
+
+This is why it’s important to monitor your site’s core web vitals and take action on low scores - and New Relic can help you do that! 
+
+## What are core web vitals?
+
+The three core web vitals are:
+* [Largest contentful paint](https://web.dev/lcp/) - measures loading performance
```
Comment (nbaenam) [inline src/content/docs/browser/new-relic-browser/guides/guide-to-core-web-vitals.mdx:23]: I think the '-' can be replaced by ':', I can't remember to see the '-' in our docs. Examples: [Transition to New Relic from Insights](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/transition-new-relic-one-insights#features), [Error tracking with errors inbox](https://docs.newrelic.com/docs/errors-inbox/errors-inbox#dates) and [in the same page](https://docs.newrelic.com/docs/errors-inbox/errors-inbox#status), or [Account settings](https://docs.newrelic.com/docs/accounts/accounts-billing/general-account-settings/intro-account-settings#user-menu)

**Review by nbaenam — COMMENTED** [review]
I've added minor comments. Great job!!

**Pull request merged**